src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -100,7 +83,7 @@ private ExpressionSyntax ParseInterpolatedStringToken()
                 // In the special case when there are no interpolations, we just construct a format string
                 // with no inserts. We must still use String.Format to get its handling of escapes such as {{,
                 // so we still treat it as a composite format string.
-                var text = Substring(originalText, openQuoteIndex + 1, closeQuoteIndex - 1);
+                var text = originalText[new Range(openQuoteIndex + 1, closeQuoteIndex)];
```

Comment:
no need for -1, Range math is sane math.

Note: we might want to move the open-quote part to be a range as well, but i haven't needed that so far.